global.yaml

```diff
@@ -72,6 +72,12 @@ global:
         cpu: 2000m
         memory: 128Mi
 
+    # proxy.interceptionControl
+    # values:
+    #  init -- the istio-init container will setup the proxy traffic interception
+    #  cni -- the CNI will setup the proxy traffic interception (use this with istio-cni)
+    interceptionControl: init
```

Comment:
@linsun what do you think of this API ? 

I like that we can extend it later.

Reply:
I'm having a problem with the sidecar inject go template trying to check if this option is set and equal to init OR if the old option istio_cni.enabled is set to false.  So, I think I should revert this new option from this PR and keep the old one and propose something new in a follow-on.